app/services/authentication_service.rb

```diff
@@ -42,15 +46,35 @@ def sign_in_user_id_from_token
   end
 
   def user_by_email
-    return if email_from_token.blank?
+    if email_from_token.blank?
+      logger.debug("No email in token")
+      return
+    end
 
     @user_by_email ||= User.find_by("lower(email) = ?", email_from_token)
+    if @user_by_email
+      logger.debug("User found from sign_in_user_id in token " + {
```

Comment:
Can we log the fact that user is found, and record ID, instead of email please, also this message is a copy and paste error. User was found using email not sign in user id